src/layers/arc-layer/arc-layer.js

```diff
@@ -240,6 +240,7 @@ export default class ArcLayer extends Layer {
           getFilterValue: gpuFilter.filterValueUpdateTriggers,
           getWidth: {
             sizeField: this.config.sizeField,
+            sizeScale: this.config.sizeScale,
```

Comment:
is this property now available on the layer side panel?

Reply:
Yes